.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+| Question           | Answer
+| ------------------ | ------------------
+| **JIRA issue**     | [EZP-XXXXX](https://jira.ez.no/browse/EZP-XXXXX)
+| **Bug fix**        | yes/no
+| **New feature**    | yes/no
+| **Target version** | `6.7` for bug fixes, `7.x` for features
```

Comment:
Maybe little change here, `6.7/6.13 for bug fixes, ...`. From time to time there is a bug which needs to be fixed in 6.13 and up, but it's not related for `6.7`.

Reply:
@kmadejski amended according to your suggestion, thanks